docs/guides/tech.md

```diff
@@ -1,120 +1,88 @@
 # Playback Technology ("Tech")
 
-Playback Technology refers to the specific browser or plugin technology used to play the video or audio. When using HTML5, the playback technology is the video or audio element. When using Flash, the playback technology is the specific Flash player used, e.g. Flowplayer, YouTube Player, video-js.swf, etc. (not just "Flash"). This could also include Silverlight, Quicktime, or any other plugin that will play back video in the browser, as long as there is an API wrapper written for it.
+Playback Technology refers to the specific browser or plugin technology used to play the video or audio.
+When using HTML5, the playback technology is the video or audio element. When using Flash and the
+[videojs-flash project](http://github.com/videojs/videojs-flash), the playback technology is the specific Flash player used,
+e.g. Flowplayer, YouTube Player, video-js.swf, etc. (not just "Flash"). This could also include Silverlight,
```

Comment:
When using Flash and videojs-flash the tech is video-js.swf.

The You Tube tech isn't a good example of a Flash-based tech any more. And Flowplayer?